src/style.css

```diff
@@ -0,0 +1,151 @@
+html {
+  font-family: Roboto, Helvetika, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  width: 200px;
+  height: 406px;
```

Comment:
Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size.
```suggestion
```